docs/guides/manage-translations.rst

```diff
@@ -0,0 +1,229 @@
+
+In this guide we will focus our efforts around two different methods: manual and using Transifex_.
+
+In both methods, we need to follow these steps to translate our documentation:
+
+#. Create translatable files (``.pot`` and ``.po`` extensions) from source language
+#. Translate the text on those files from source language to target language
+#. Build the documentation in *target language* using the translated texts
+
+Besides these steps, once we have published our first translated version of our documentation,
+we will want to keep it updated from the source language. At that time, the workflow would be:
+
+#. Update our translatable files from source language
+#. Translate only *new* and *modified* texts in source language into target language
+#. Build the documentation using the most up to date translations
+
+
+Create translatable files
+-------------------------
+
+To generate these ``.pot`` files it's needed to run this command from your ``docs/`` directory:
+
+.. code-block:: console
+
+   $ sphinx-build -b gettext . _build/gettext
+
+.. tip::
+
+   We recommend configuring Sphinx to use :ref:`sphinx:std:confval:gettext_uuid` as ``True``
+   and also :ref:`sphinx:std:confval:gettext_compact` as ``False`` to generate ``.pot`` files.
+
+
+This command will leave the generated files under ``_build/gettext``.
+
+
+Translate text from source language
+-----------------------------------
+
+Manually
+~~~~~~~~
+
+We recommend using `sphinx-intl`_ tool for this workflow.
+
+.. _sphinx-intl: https://pypi.org/project/sphinx-intl/
+
+First, we need to install it:
+
+.. code-block:: console
+
+   $ pip install sphinx-intl
+
+
+As a second step, we want to create a directory with each translated file per target language
+(in this example we are using Spanish/Argentina and Portuguese/Brazil).
+This can be achieved with the following command:
+
+.. code-block:: console
+
+   $ sphinx-intl update -p _build/gettext -l es_AR -l pt_BR
+
+This command will create a directory structure similar to the following
+(with one ``.po`` file per ``.rst`` file in your documentation)::
+
+  locale
+  ├── es_AR
+  │   └── LC_MESSAGES
+  │       └── index.po
+  └── pt_BR
+      └── LC_MESSAGES
+          └── index.po
+
+
+Now, you can just open those ``.po`` files with a text editor and translate them taking care of no breaking the reST notation.
+Example:
+
+.. code-block:: po
+
+   # b8f891b8443f4a45994c9c0a6bec14c3
+   #: ../../index.rst:4
+   msgid ""
+   "Read the Docs hosts documentation for the open source community."
+   "It supports :ref:`Sphinx <sphinx>` docs written with reStructuredText."
+   msgstr ""
+   "FILL HERE BY TARGET LANGUAGE FILL HERE BY TARGET LANGUAGE FILL HERE "
+   "BY TARGET LANGUAGE :ref:`Sphinx <sphinx>` FILL HERE."
+
+
+Using Transifex
+~~~~~~~~~~~~~~~
+
+Transifex_ is a platform that simplifies the manipulation of ``.po`` files and offers many useful features to make the translation process as smooth as possible.
```

Comment:
I guess given we're about to make a nice advertising for Transifex services (which is fine by me), it would be fair to mention the competition and their relationship with open-source projects.

Transifex:
> We offer Transifex for free to Open Source projects that have no funding, revenue, and/or commercialization model
https://docs.transifex.com/projects/open-source-project

Crowdin:
Free too, [with conditions](https://crowdin.com/page/open-source-project-setup-request)

Zanata:
Free, open source, self-hostable (by Red Hat), but as far as I know, less features
http://zanata.org/

And then there's PhraseApp, Smartling and many others.

Reply:
Good point.

This was something that bothered to me while I was writing this guide. The question was: "Why we are suggesting a non-free solution?" I read that note that you mention here, but it still made me noise at that time.

I commented this to @agjohnson and he said that maybe it does worth to have multiples guides instead just one. Like:

* How to use Transifex and Read the Docs?
* How to use Crowdin and Read the Docs?
* etc

I think this is the direction that I would like to follow, but I do not have experience with these other tools and this will take me some time to read, test the flow and write the docs. So, I'd say that we could have a initial guide (as this one) and work in the other guides soon.

Maybe, we could mention the other solutions (name and link) and add a note saying that we are working on writing other guides for them and contributions are welcomed :)

What do you think?

Reply:
Wow, you seem way more ambitious than me :D I would tend to just mention names and links as you said, and maybe a link to every project documentation.

If those projects have documentation, then they might be more precise and up to date than ours will be (given we're really following a classic use case here).

If those projects don't have documentation... Do we really want to link to them ? :D

That being said, I'm totally ok keeping the transifex doc, especially now it's been written, because if there's at least one doc, then it's easier to understand what we're trying to do and adapt it to another tool.

Lastly, there's another alternative if you want to explore it : a "logical" documentation that would explain the steps and you can see examples with different tools. Something like

Upload the sources:
[with transifex](#) | [with crowdin](#) | ...
[panel that would change depending on the click above]
 
This would maybe help people understand that whatever the tools, the steps are the same.